Microsoft.Toolkit.Uwp.UI.Controls.Design/Controls/RadialGauge.Typedata.cs

```diff
@@ -0,0 +1,37 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Microsoft.Toolkit.Uwp.UI.Controls.Design
+    internal static partial class ControlTypes
```

Comment:
```suggestion
namespace Microsoft.Toolkit.Uwp.UI.Controls.Design
{
    internal static partial class ControlTypes
```

Reply:
@Nirmal4G can you re-check the 'Allow edits by maintainers' option? This would have allowed me to commit this fix and see if we got a successful CI build. Thanks! It'll only grant access to your own branch in the fork.

Reply:
I did fix it when the build failed but it seems that I forgot to push the changes. Since I got it build locally, I didn't bother to check the CI. Sorry about that!!

> GitHub should have failed status check notifications.

Reply:
@michael-hawker Sure, I enabled it.